odbc/src/api/errors.rs

```diff
@@ -12,132 +12,132 @@ use thiserror::Error;
 #[derive(Debug, Error, Clone)]
 #[repr(C)]
 pub enum ODBCError {
-    #[error("[{}][API] {0}", VENDOR_IDENTIFIER)]
+    #[error("[{vendor}][API] {0}", vendor = VENDOR_IDENTIFIER)]
     General(&'static str),
-    #[error("[{}][API] {0}", VENDOR_IDENTIFIER)]
+    #[error("[{vendor}][API] {0}", vendor = VENDOR_IDENTIFIER)]
     GeneralWarning(String),
-    #[error("[{}][API] Caught panic: {0}", VENDOR_IDENTIFIER)]
+    #[error("[{vendor}][API] Caught panic: {0}", vendor = VENDOR_IDENTIFIER)]
```

Comment:
`thiserror` versions >= 2.0.0 throw an ambiguous reference error unless I change `VENDOR_IDENTIFIER` to a named argument. This is the only change in this file.

Reply:
Odd